### PR TITLE
James Friedman Response

### PR DESCRIPTION
**Pull request by jpfriedman18**

### CODE REVIEW

**Comment by jpfriedman18** [comment]
Forgot to add code to diagnostic.rb!

File: app/models/licensee.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Licensee < ActiveRecord::Base
+end
```
Comment (GA-MEB) [inline app/models/licensee.rb:2]: Model : Check

File: db/migrate/20160219140826_create_licensees.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      t.integer :height
+      t.integer :weight
+      t.date :born_on
+      t.string :eye_color
+      t.string :hair_color
+
+      t.timestamps null: false
+    end
+  end
+end
```
Comment (GA-MEB) [inline db/migrate/20160219140826_create_licensees.rb:16]: Migration : Check

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    t.integer  "height"
+    t.integer  "weight"
+    t.date     "born_on"
+    t.string   "eye_color"
+    t.string   "hair_color"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+end
```
Comment (GA-MEB) [inline db/schema.rb:32]: Ran Migration : Check

File: lib/tasks/db.rake [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  task licensees: :environment do
+    Licensee.transaction do
+      CSV.foreach(Rails.root + 'data/licensees.csv', headers: true) do |row|
+        licensee = row.to_hash
+        Licensee.create! licensee unless Licensee.exists? licensee
+      end
+    end
+  end
+end
+end
```
Comment (GA-MEB) [inline lib/tasks/db.rake:15]: Extra `end`, otherwise right.

File: scripts/create_person.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+caren = Licensee.create surname: 'Bishop',
+                      given_name: 'Caren',
+                      gender: 'f',
+                      height: 63,
+                      weight: 132,
+                      born_on: '1943-09-26',
+                      eye_color: 'Brown',
+                      hair_color: 'Black'
+
+p caren
```
Comment (GA-MEB) [inline scripts/create_person.rb:10]: Correct.

Comment (GA-MEB) [inline scripts/create_person.rb:10]: Also, while it contradicts the directions, I like your instinct of keeping scripts separate and putting them in a separate directory.

**Comment by GA-MEB** [comment]
3 / 6 (No answer for Q4, Q5, Q6)

**Pull request closed without merging**